fabtests/benchmarks/benchmark_shared.c

```diff
@@ -70,6 +70,12 @@ void ft_parse_benchmark_opts(int op, char *optarg)
 	case 'W':
 		opts.window_size = atoi(optarg);
 		break;
+	case 'T':
+		/* Reuse FT_OPT_SKIP_MSG_ALLOC as a flag to post transmits
+		 * before receives.
+		 */
+		opts.options |= FT_OPT_SKIP_MSG_ALLOC;
```

Comment:
As mentioned elsewhere, use a dedicated flag for this.

Reply:
Will define FT_OPT_NO_PRE_POSTED_RX